draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -790,7 +792,7 @@ Further details of the algorithm are given in the following sections.
 
 ## Message identifiers
 
-### Primary ID (required)
```

Comment:
OK, I've reinstated the (required) and (optional) in the section titles and hand-specified the reference target.